src/main/java/com/useful/ucars/uCarsListener.java

```diff
@@ -2190,4 +2189,8 @@ public void init() {
 	public List<String> getWorldList() {
 		return ucarworlds;
 	}
+	
+	public boolean getMultiverse() {
```

Comment:
"isMultiverseEnabled" would be much clearer imho, or at the least "isMultiverse". (is convention for booleans)